src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNT = 9;

```

Comment:
What if someone adds one more color to enum? This class will become incorrect.
```suggestion
    private static final int COLOR_COUNT = Color.values().length;
```